content/en/docs/setup/install/multicluster/shared/index.md

```diff
@@ -203,8 +271,16 @@ spec:
         clusterName: ${REMOTE_CLUSTER_NAME}
       network: ${REMOTE_CLUSTER_NETWORK}
 
-      # Replace ISTIOD_REMOTE_EP with the the value of ISTIOD_REMOTE_EP set earlier.
+      # Replaces ISTIOD_REMOTE_EP with the the value of ISTIOD_REMOTE_EP set earlier.
       remotePilotAddress: ${ISTIOD_REMOTE_EP}
+
+  ## The ingressgateway is not required in the remote cluster if both clusters are on
+  ## the same network. Uncomment the lines below to disable the ingress-gateway component.
```

Comment:
Reverse the clause order: "To disable the ingress-gateway component, uncomment the lines below."

Reply:
```suggestion
  ## the same network. To disable the istio-ingressgateway component, uncomment the lines below.```